### ISSUE TITLE
Accessibility Zoom doesn't work mac to mac

### ISSUE THREAD

**Issue by marcobazzani** [debauchee/barrier#1063]
I have Ctrl+wheel used to zoom the screen,
on the guest system this command doesn't work with barrier mouse+keyboard while it works with guest mouse+kb

**Comment by Tyler-Keith-Thompson** [comment]
BUMP. This issue interrupts my workflow, too.

**Comment by briantully** [comment]
I'm having this same issue. It's a shame it's not supported.

By the way what does "while it works with guest mouse+kb" mean? Is "guest" an alternative to "barrier"?

**Comment by marcobazzani** [comment]
> I'm having this same issue. It's a shame it's not supported.
> 
> By the way what does "while it works with guest mouse+kb" mean? Is "guest" an alternative to "barrier"?

guest is the secondary computer anyway I bought a Logitech MX Key+ Mouse witch always works
